application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1765,6 +1808,50 @@ type RealTimeEstimate {
   time: OffsetDateTime!
 }
 
+"""
+Timing of an arrival or a departure to or from a stop. May contain real-time information if
+available. This is used when there is a known scheduled time.
+"""
+type RegularRealTimeArrivalDepartureTime {
+  "The estimated time of the event. If no real-time information is available, this is null."
+  estimated: RealTimeEstimate
+  "The scheduled time of the event."
+  scheduledTime: OffsetDateTime!
+}
+
+"""
+Regular real-time stop time represents the time when a specific trip on a specific date arrives to and/or departs from a specific stop.
+This can include real-time estimates.
+"""
+type RegularRealTimeStopTime {
```

Comment:
Name: `StopCall`?

Reply:
I'm ok with using the word call, but we need to make a final decision on this prefix vs no-prefix discussion, I've made a more detailed comment on https://github.com/opentripplanner/OpenTripPlanner/pull/5393#discussion_r1843767697